programs/cleos/main.cpp

```diff
@@ -1003,10 +1001,8 @@ struct create_account_subcommand {
       createAccount->add_option("ActiveKey", active_key_str, localized("The active public key or permission level for the new account"));
 
       if (!simple) {
-         createAccount->add_option("--stake-net", stake_net,
-                                   (localized("The amount of tokens delegated for net bandwidth")))->required();
-         createAccount->add_option("--stake-cpu", stake_cpu,
-                                   (localized("The amount of tokens delegated for CPU bandwidth")))->required();
+         createAccount->add_option("--stake-quantity", stake_quantity,
```

Comment:
**--stake** should be fine. No need to add **-quantity** here

Reply:
changed